apps/web/src/setup-button.tsx

```diff
@@ -1,7 +1,7 @@
 import detectEthereumProvider from '@metamask/detect-provider'
 import { useAsyncCallback } from 'react-use-async-callback'
 import Image from 'next/image'
-import { inter } from '../pages/index'
+import { inter } from '../pages/[chain]'
```

Comment:
hey this wasnt you but it is weird to import from a page into a component file. I must have messed it up. if there is another place to import inter from would be great to change that

Reply:
"Hey, while you're at it...."